mesonbuild/envconfig.py

```diff
@@ -140,6 +140,9 @@ def get_root(self) -> T.Optional[T.Union[str, T.List[str]]]:
     def get_sys_root(self) -> T.Optional[T.Union[str, T.List[str]]]:
         return self.properties.get('sys_root', None)
 
+    def get_pkg_config_libdir(self) -> T.Optional[T.Union[str, T.List[str]]]:
+        return self.properties.get('pkg_config_libdir', None)
```

Comment:
Could we ensure this is not a plain str (either None or a List[str]), since I'm pretty sure that plain str is broken anyway? something like:
```python
def get_pkg_config_libdir(self) -> T.Optional[T.List[str]]:
    p = self.properties.get('pkg_config_libdir', None)
    if p is None:
         return p
    return mesonlib.listify(p)
```

Reply:
indeed, better listify here rather than above.

Reply:
the same, please clarify what i need to listify ;)

Reply:
@dabrain34 just like the snippet from @dcbaker ;-)

Reply:
sorry i missed it :)